src/main/java/com/datapipe/jenkins/vault/configuration/VaultConfiguration.java

```diff
@@ -30,21 +30,25 @@
 
     private boolean skipSslVerification = false;
 
+    private String vaultNamespace;
+
     public VaultConfiguration() {
         // no args constructor
     }
 
     @DataBoundConstructor
-    public VaultConfiguration(String vaultUrl, String vaultCredentialId, boolean failIfNotFound) {
+    public VaultConfiguration(String vaultUrl, String vaultCredentialId, boolean failIfNotFound, String vaultNamespace) {
```

Comment:
You can remove it from the `@DataBoundConstructor` since you already created a `@DataBoundSetter`
```suggestion
    public VaultConfiguration(String vaultUrl, String vaultCredentialId, boolean failIfNotFound) {
```
Also this breaks binary compatibility :sweat: